bgpd/bgp_zebra.c

```diff
@@ -3603,24 +3611,43 @@ static int bgp_zebra_srv6_sid_notify(ZAPI_CALLBACK_ARGS)
 		return -1;
 	}
 
+	locator_bgp = bgp_srv6_locator_lookup(bgp_vrf, bgp_get_default());
+	if (!locator_bgp)
+		return -1;
+
 	/* Handle notification */
 	switch (note) {
 	case ZAPI_SRV6_SID_ALLOCATED:
 		if (BGP_DEBUG(zebra, ZEBRA))
 			zlog_debug("SRv6 SID %pI6 %s : ALLOCATED", &sid_addr,
 				   srv6_sid_ctx2str(buf, sizeof(buf), &ctx));
 
+		if (strcmp(locator_bgp->name, loc_name) != 0) {
+			zlog_err("%s(%d): %s, SRv6 Locator name unmatch %s:%s, releasing SID.",
```

Comment:
Below we use zlog_debug before returning -1, here zlog_err, any reason it's special here?